dev/CommandBarFlyout/InteractionTests/TextCommandBarFlyoutTests.cs

```diff
@@ -480,11 +475,19 @@ public void ValidateKeyboarding()
                 {
                     Log.Comment("Use Escape to close the context menu. The TextBox should now have focus.");
                     KeyboardHelper.PressKey(Key.Escape);
+
+                    // On 19H1, there's a bug with the focus restoration code, so we'll manually restore focus to allow the rest of the test to run.
```

Comment:
>On 19H1, there's a bug with the focus restoration code, so we'll manually restore focus to allow the rest of the test to run. [](start = 23, length = 125)

Is there a bug number we can add to this comment to track it a little tighter?

Reply:
It seems as though it's already fixed in the latest version - I can't reproduce it there.  So it should be good to just leave this comment here.